src/components/style/navbar.css

```diff
@@ -0,0 +1,501 @@
+.FDSnavfeaturedItemImageContainer {
+  box-shadow: -0.625em 0.625em 0 0 rgb(0 0 0 / 50%);
+  width: 300px;
+  height: 181px;
+}
+
+.FDSnavfeaturedItemImage {
+  display: block;
+}
+
+.FDSnavfeaturedItemText {
+  margin-top: var(--space-l);
+  font-size: 1.25em;
+  color: var(--color-white);
+  font-weight: 900;
+}
+
+.FDSnavfeaturedItemLearnMore {
+  margin-top: var(--space-l);
+  color: var(--color-white);
+  font-size: 1em;
+  font-weight: 700;
+}
+
+.FDSnavfeaturedItemLearnMore:hover {
+  text-decoration: underline;
+}
+
+.FDSnavfeaturedItem {
+  cursor: auto;
+  position: absolute;
+  top: 0;
+  right: 0;
+  margin: 0 !important;
+  padding: 35px 35px 0 0 !important;
+  background-color: transparent !important;
+}
+
+.FDSnaviconButton {
+  display: block;
+}
+
+.FDSnavmobileIconButton {
+  display: none;
+}
+
+.FDSnavtopContainer {
+  display: flex;
+}
+
+.FDSnavcontainer {
+  display: flex;
+  position: relative;
+  align-items: center;
+  max-width: 1300px;
+  margin: auto;
+}
+
+.FDSnavhasSubMenu {
+  width: 900px;
+  height: 350px;
+  right: calc(100% - 40em) !important;
+}
+
+.FDSnavdropdownContainer {
+  opacity: 0;
+  visibility: hidden;
+  position: absolute;
+  display: flex;
+  flex-direction: row;
+  right: -5em;
+  top: 100%;
+  z-index: calc(var(--zindex-navigation) + 1);
+}
+
+.FDSnavmenuContainer > li > a {
+  padding-right: 30px;
+  padding-left: 30px;
+  width: 263px;
+}
+
+.FDSnavmenuContainer {
+  background-color: #a02385;
+  padding: var(--space-l) 0;
+  width: 100%;
+}
+
+.FDSnavsubMenuList {
+  padding: 0;
+  width: 100%;
+  background-color: #6a1e6d;
+}
+
+.FDSnavsubMenuContainer {
+  padding: var(--space-l) 0;
+  display: none;
+  position: absolute;
+  top: 0;
+  left: 263px;
+  width: calc(100% - 263px);
+  height: 100%;
+  background-color: #6a1e6d;
+}
```

Comment:
I'm not sure what's wrong here, but when I run storybook locally the submenu is not centered

Reply:
The submenu shouldn't be centered with the button.

Reply:
i think i used the wrong word when i said 'centered' - i just mean it doesn't fit inside the menu

Reply:
<img width="1272" alt="Screen Shot 2022-04-19 at 4 37 35 PM" src="https://user-images.githubusercontent.com/59481400/164091513-5daa8721-29e8-4d7f-b5f0-72751f0fd55a.png">

Reply:
<img width="1172" alt="Screen Shot 2022-04-19 at 4 38 27 PM" src="https://user-images.githubusercontent.com/59481400/164091634-3a9dcc0b-5806-414b-b9ba-1b6d0d1b0b5b.png">

Reply:
maybe just has to do with the svg not rendering

Reply:
Oh good catch. I had to update the box-sizing.